### DISCUSSION TITLE
Using Gin or Gist to index TEXT column

### DISCUSSION THREAD

**Discussion by ygrenzinger** [prisma/prisma#16479] [Q&A] [answered]
Hello,

I want to do "ILIKE" query on a text column (String in schema definition).

I have done some tests following this article https://www.cybertec-postgresql.com/en/postgresql-more-performance-for-like-and-ilike-statements/
And Gist index offers the best perf.
The SQL to enable it looks like this 

`CREATE INDEX idx_brandaccount_name_gist ON "public"."BrandAccount" USING gist (name gist_trgm_ops);` 

But it doesn't seem possible to do it in schema :(
Why not ? missing feature ?
Is it possible to add the SQL directly in the migration script ?

Thanks

**Comment by ygrenzinger** [comment, marked as answer]
Oh seems i finally found a solution after a good night :D
` @@index(fields: [name(ops: raw("gin_trgm_ops"))], type: Gin)`

**Comment by ludralph** [comment]
Hi there,

To keep our discussions organized and focused on the most relevant topics, we’re reviewing and tidying up our backlog. As part of this process, we’re closing discussions that have already been marked as answered but remain open.

If this discussion still requires further input or clarification, feel free to reopen it or start a new one with updated details. Your contributions are invaluable to the community, and we’re here to help!

For more details about our priorities and vision for the future of Prisma ORM, check out our latest blog post: https://www.prisma.io/blog/prisma-orm-manifesto.

Thank you for your understanding and ongoing support of the Prisma community!

**Comment by anuuu16** [comment]
Hello all,
I have prepared a blog for adding Full-Text Search (FTS) in Prisma with PostgreSQL tsvector, Without Migration Drift

https://medium.com/@chauhananubhav16/bulletproof-full-text-search-fts-in-prisma-with-postgresql-tsvector-without-migration-drift-c421f63aaab3
